core/src/test/java/org/apache/iceberg/TestMetricsModes.java

```diff
@@ -19,22 +19,38 @@
 
 package org.apache.iceberg;
 
+import java.io.File;
 import java.util.Map;
 import org.apache.iceberg.MetricsModes.Counts;
 import org.apache.iceberg.MetricsModes.Full;
 import org.apache.iceberg.MetricsModes.None;
 import org.apache.iceberg.MetricsModes.Truncate;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.types.Types;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import static org.apache.iceberg.types.Types.NestedField.required;
 
 public class TestMetricsModes {
 
   @Rule
   public ExpectedException exceptionRule = ExpectedException.none();
 
+  @Rule
+  public TemporaryFolder temp = new TemporaryFolder();
+
+  private static final int FORMAT_V2 = 2;
```

Comment:
nit: Should we have it defined as public constant in a utils files, so that it can be referenced in future?

Reply:
I actually made this a parameter as per the comment above , but if we don't do it that way I can look at this.